portal/views/patients.py

```diff
@@ -99,8 +99,16 @@ def patients_root():
                      UserIntervention.intervention_id.in_(ui_list)))
         patients = patients.union(ui_patients)
 
+    # get assessment status only if it is needed as specified by config
+    if 'status' in current_app.config.get('PATIENT_LIST_ADDL_FIELDS'):
+        patient_list = []
+        for patient in patients:
+            patient.assessment_status = overall_assessment_status(patient.id)
+            patient_list.append(patient)
+        patients = patient_list
+
```

Comment:
I'm not sure I grok the pattern of creating a secondary list `patient_list` and then copying that back over the top of `patients` after iteration.  I don't believe you need to use `patient_list` here at all - unless I'm missing something?